lib/stack.rb

```diff
@@ -0,0 +1,36 @@
+# - Create a Stack class which implements methods to push, pop,
+# and peek at the top item.
+# - Add an empty? helper.
+# - Verify that you can reverse a string with your stack:
+
+class Stack
+  attr_accessor :container
+
+  def initialize(cont = [])
+    @container = cont
+  end
```

Comment:
you can also store the size of the stack as an instance variable and use `@size` in `push`, `pop` methods instead of length.